jfr-container-action/init.groovy

```diff
@@ -0,0 +1,5 @@
+import hudson.model.DownloadService;
+println 'Downloading the plugins meta info...'
+for (d in DownloadService.Downloadable.all()) {
+    d.updateNow();
```

Comment:
Will this update all installed plugins to their latest versions?

Reply:
No, it won't. Based on my experiments, it only updates the [periodically updated JSON data file](https://updates.jenkins.io/updates/). This behavior is the same as the description specified in [its doc](https://javadoc.jenkins.io/hudson/model/DownloadService.Downloadable.html).

Reply:
You can check your own $JENKINS_HOME/updates directory. This script will help update meta info stored in this directory.